actors/builtin/miner/miner_test.go

```diff
@@ -892,11 +1023,40 @@ func makeProveCommit(sectorNo abi.SectorNumber) *miner.ProveCommitSectorParams {
 	}
 }
 
+func makePowerClaimUpdate(rawDelta, qaDelta abi.StoragePower) *power.UpdateClaimedPowerParams {
+	return &power.UpdateClaimedPowerParams{
+		RawByteDelta:         rawDelta,
+		QualityAdjustedDelta: qaDelta,
+	}
+}
+
+func makeFaultParamsFromFaultingSectors(t testing.TB, st *miner.State, store adt.Store, faultSectorInfos []*miner.SectorOnChainInfo) *miner.DeclareFaultsParams {
+	deadlines, err := st.LoadDeadlines(store)
+	require.NoError(t, err)
+	faultAtDeadline := make(map[uint64][]uint64)
+	for _, sectorInfo := range faultSectorInfos {
```

Comment:
Please add an explanatory comment, like "find the deadline for each faulty sector, which must be provided with the fault declaration".